packages/link-modules/src/services/dynamic-service-class.ts

```diff
@@ -0,0 +1,18 @@
+import { LinkModuleService } from "@services"
+import { ModuleJoinerConfig } from "@medusajs/types"
+
+export function getModuleService(joinerConfig: ModuleJoinerConfig) {
+  return class LinkService extends LinkModuleService<unknown> {
+    __joinerConfig(): ModuleJoinerConfig {
+      return joinerConfig as ModuleJoinerConfig
+    }
+  }
+}
+
+export function getReadOnlyModuleService(joinerConfig: ModuleJoinerConfig) {
```

Comment:
comment: I am not sure of the ReadOnly wording, it makes you thing that if it is readonly you could only retrieve, list and if it is not then you can write. but it is readonly, we don't even have list and retrieve no? Or maybe I ve missed it